ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -29,6 +29,47 @@ def check_pod_vanished(pod_names):
             raise UnexpectedBehaviour
 
 
+def delete_logging_namespaces(force=False):
+    """
+    Deleting namespaces
+    1. Openshift-operators-redhat
+    2. Openshift-logging
+
+    """
+    openshift_logging_namespace = ocp.OCP(
+        kind=constants.NAMESPACES, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+    )
+    openshift_operators_redhat_namespace = ocp.OCP(
+        kind=constants.NAMESPACES,
+        namespace=constants.OPENSHIFT_OPERATORS_REDHAT_NAMESPACE,
+    )
+
+    if openshift_operators_redhat_namespace.get():
+        try:
+            openshift_operators_redhat_namespace.delete(
+                resource_name=constants.OPENSHIFT_OPERATORS_REDHAT_NAMESPACE,
+                force=force,
+            )
+            openshift_operators_redhat_namespace.wait_for_delete()
+            logger.info(
+                "The project openshift-opertors-redhat got deleted successfully"
+            )
+        except CommandFailed as e:
+            assert "not found" in str(e)
```

Comment:
IIUC you're trying to assert if the exception is other than "not found" right? Its better to log the e.args[0] as assert failure message so that we know what is the exception when it fails.

Reply:
better to implement this also like my next comment below :point_down:

Reply:
I removed the assert statement for "not found" but can't raise commandFailed as you suggested reason: if openshift-operators-redhat project deletion fails, it might not delete openshift-logging project. 
The only exception we expect here is "Project not Found/exists" as we are just trying to delete the project here.